docs/standard/base-types/standard-numeric-format-strings.md

```diff
@@ -194,7 +194,12 @@ Standard numeric format strings are used to format common numeric types. A stand
  However, if the number is a <xref:System.Decimal> and the precision specifier is omitted, fixed-point notation is always used and trailing zeros are preserved.  
   
  If scientific notation is used, the exponent in the result is prefixed with "E" if the format specifier is "G", or "e" if the format specifier is "g". The exponent contains a minimum of two digits. This differs from the format for scientific notation that is produced by the exponential format specifier, which includes a minimum of three digits in the exponent.  
-  
+ 
+Note that, when used with a <xref:System.Double> value, the "G17" format specifier ensures that the original <xref:System.Double> value successfully round-trips. This is because internally the runtime stores a <xref:System.Double> value with 17 digits of precision. We recommend its use instead of the ["R" format specifier](#RFormatString), since in some cases "R" fails to successfully round-trip double values. The following example illustrates one such case.
```

Comment:
> This is because internally the runtime stores a <xref:System.Double> value with 17 digits of precision.

Possibly:
> This is because <xref:System.Double> is an IEEE 754:2008 compliant double-precision floating-point number which gives up to 17 significant decimal digits of precision

Note to the above suggestion: The `IEEE 754:2008 double-precision floating-point` format is formally called `binary64`

Reply:
@tannergooding, `Single` compiles with the standard as well (the binary32 format), no?

Reply:
Yes, and gives up to 9 significant decimal digits of precision.